float64[] sonar_scan_1 # we assume values between 0 and 1
float64[] sonar_scan_2 # we assume values between 0 and 1
int32 dimension_size # in 3D: NxNxN then just give N
float64 size_of_voxel
float64 potential_for_necessary_peak  # default mostly 0.1 Higher values less peaks, lower values more peaks
geometry_msgs/Pose initial_guess # 3D
bool use_clahe
bool debug
bool timing_computation_duration
---
fsregistration/PotentialSolution3D potential_solution # always calculates the closest peak of the initial guess